utils/config_manager.py

```
@@ -247,7 +247,11 @@ def migrate_catgirl_reserved(catgirl_data: dict) -> bool:
         legacy_keys=("idleAnimation",),
     )
     if idle_animation:
-        changed |= set_reserved(catgirl_data, "avatar", "vrm", "idle_animation", str(idle_animation))
+        # 向前兼容：旧版存的是 string，迁移为 list
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)